packages/backend/src/peripherals/database/migrations/018_aggregate_reports.ts

```diff
@@ -0,0 +1,29 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('aggregate_reports', function (table) {
+    table.bigInteger('unix_timestamp').notNullable()
```

Comment:
Add index (unless Postgesql creates indexes for the fields separately when the public key is a composite)

Reply:
Looks like adding an index to just `project_id` cover all the cases (https://devcenter.heroku.com/articles/postgresql-indexes#multi-column-indexes)